addRate.js

```diff
@@ -35,4 +35,14 @@ $(document).ready(function() {
       },
     },
   });
+
+  user = getLoggedUser();
```

Comment:
what happens when API responses with an error here? for example when session has timed out?